tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -142,6 +166,119 @@ class OfstedPage {
         this.elements.currentRatings.NoDataMessage().should('be.visible');
         return this;
     }
+
+    ///previous ratings///
+
+    public checkOfstedPreviousRatingsPageHeaderPresent(): this {
+        this.elements.subpageHeader().should('contain', 'Previous ratings');
+        return this;
+    }
+
+    public checkOfstedPreviousRatingsTableHeadersPresent(): this {
+        this.elements.previousRatings.SchoolNameHeader().should('be.visible');
+        this.elements.previousRatings.qualityOfEducationHeader().should('be.visible');
+        this.elements.previousRatings.behaviourAndAttitudesHeader().should('be.visible');
+        this.elements.previousRatings.personalDevelopmentHeader().should('be.visible');
+        this.elements.previousRatings.leadershipAndManagementHeader().should('be.visible');
+        this.elements.previousRatings.earlyYearsProvisionHeader().should('be.visible');
+        this.elements.previousRatings.sixthFormProvisionHeader().should('be.visible');
+        this.elements.previousRatings.beforeOrAfterJoiningHeader().scrollIntoView();
+        this.elements.previousRatings.beforeOrAfterJoiningHeader().should('be.visible');
+        return this;
+    }
+
+    public checkOfstedPreviousRatingsSorting(): this {
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.SchoolName,
+            this.elements.previousRatings.SchoolNameHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.qualityOfEducation,
+            this.elements.previousRatings.qualityOfEducationHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.behaviourAndAttitudes,
+            this.elements.previousRatings.behaviourAndAttitudesHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.personalDevelopment,
+            this.elements.previousRatings.personalDevelopmentHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.leadershipAndManagement,
+            this.elements.previousRatings.leadershipAndManagementHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.earlyYearsProvision,
+            this.elements.previousRatings.earlyYearsProvisionHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.sixthFormProvision,
+            this.elements.previousRatings.sixthFormProvisionHeader
+        );
+        TableUtility.checkStringSorting(
+            this.elements.previousRatings.beforeOrAfterJoining,
+            this.elements.previousRatings.beforeOrAfterJoiningHeader
+        );
+        return this;
+    }
+
+    public checkPreviousRatingsQualityOfEducationJudgementsPresent(): this {
+        this.elements.previousRatings.qualityOfEducation().each((element) => {
+            const text = element.text().trim();
```

Comment:
The text shouldn't have additional spaces so if we trim it then we will miss any bugs in this area

Reply:
Trims now removed